test/fixtures/php/drupal10FilePathToContent.json

```diff
@@ -0,0 +1,100 @@
+{
```

Comment:
nit: could we come up with a simpler example, so we can see what the problem in this file is?

Reply:
I tried to cut it down, but it didn't parse correctly. I don't think it's worth the effort